sw-precache-config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  staticFileGlobs: [
+    'out/*.html',
+    'out/static/*',
```

Comment:
This configuration does not cache the font files and images.

![image](https://cloud.githubusercontent.com/assets/193136/26245722/a5c1426c-3cbf-11e7-8b8a-bc9ee2deb12f.png)

Use `out/static/**/*` instead.